protocol/x/clob/abci.go

```diff
@@ -136,10 +136,6 @@ func PrepareCheckState(
 		log.BlockHeight, ctx.BlockHeight()+1,
 	)
 
-	// TODO(CORE-538): Is there a cleaner way to check to see if this is genesis?
```

Comment:
I added this genesis check to get some tests working during the upgrade to cosmos 0.50 but it isn't necessary.